resources/ory/templates/virtualservice-hydra.yaml

```diff
@@ -1,7 +1,10 @@
+{{- if .Capabilities.APIVersions.Has "networking.istio.io/v1alpha3" }}
+{{- if .Values.hydra.service.public.enabled -}}
+---
 apiVersion: networking.istio.io/v1alpha3
```

Comment:
Probably should be in the templates of hydra?

Reply:
no, hydra by itself doesn't depend on istio, our ory chart does. Hence it is in the ory chart